test/stream_execution_test.cpp

```diff
@@ -97,7 +97,15 @@ TEST_CASE(test1)
     p.add_instruction(migraphx::op::concat{1}, p1, p2);
     p.compile(stream_execution_target{});
     CHECK(std::count_if(
-              p.begin(), p.end(), [](auto&& ins) { return ins.name() == "gpu::set_stream"; }) == 3);
+              p.begin(), p.end(), [](auto&& ins) { return ins.name() == "set_stream"; }) == 3);
+    CHECK(std::count_if(p.begin(), p.end(), [](auto&& ins) { return ins.get_stream() == 0; }) == 2);
```

Comment:
Why do you use `count_if` here? You should still check which instruction is scheduled on which stream(ie `CHECK(in1->get_stream() == 1)`)?